backend/game/player/player.py

```diff
@@ -7,13 +7,25 @@
 
 class Player:
     def __init__(self):
+        self.monster_name = None
+        self.username = None
+        self.password = None
         self.maximum_health = self.current_health = constants.DEFAULT_HEALTH
         self.location = Locations.OUTSIDE
         self.is_alive = True
         self.victory_points = constants.DEATH_HIT_POINT
         self.energy = constants.DEFAULT_ENERGY_CUBE
         self.cards: List[Card] = []
 
+    def set_monster_name(self, monster_name):
+        self.monster_name = monster_name
+
+    def set_username(self, username):
+        self.username = username
+
+    def set_password(self, password):
```

Comment:
@otaiwo70  The class itself still shouldn't store plaintext passwords. The entire gamestate is accessible to other players, including the Player classes. I would suggest not putting password in any of these three pull requests for now and adding it later.

Reply:
password reference removed